src/apps/backend/modules/account/internal/store/account-db.ts

```diff
@@ -15,6 +16,11 @@ export const AccountDbSchema: Schema = new Schema<AccountDB>(
       type: Boolean,
       required: true,
     },
+    contactNumber: {
+      type: String,
```

Comment:
`contactNumber` should not be a string but a type that stores both country code and phone number separately